Assets/Scripts/Player/PlayerInteractSystem.cs

```diff
@@ -0,0 +1,74 @@
+/*****************************************************************************
+// File Name :         PlayerInteractSystem.cs
+// Author :            Mark Hanson
+// Creation Date :     5/22/2024
+//
+// Brief Description : Class used for detecting interactable objects 
+*****************************************************************************/
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerInteractSystem
```

Comment:
Is this class still needed? It seems to be redundant with Marissa's interaction system.

Reply:
Just old interaction system I'm sure this one isn't used anymore.